### ISSUE TITLE
flesh out public documentation further

### ISSUE THREAD

**Issue by colemickens** [MinisculeGirraffe/oidc_jwt_validator#8]
I can also take a crack at this, might help with some folks' smell-test and likelihood for adoption

**Comment by MinisculeGirraffe** [comment]
Yeah agree. Any additions you can add would be super appreciated.

I've been meaning to go back flesh out some more examples also.  Real life job just eats into my open source time.

**Comment by colemickens** [comment]
Ah, my apologies, I think I'm going in a different direction and probably won't end up doing this, sorry. Figured I'd mention it in  case someone else wants to pick this up. (Otherwise of course it can be closed.)

**Comment by MinisculeGirraffe** [comment]
No worries man. 

Good luck with whatever you're working on!
